pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -517,7 +517,7 @@ def _initialize_deepspeed_train(self, model):
         self.model = model
 
     @contextlib.contextmanager
-    def model_sharded_context(self) -> Generator[None, None, None]:
+    def model_sharded_context(self) -> Generator:
```

Comment:
I think the annotation is needed, as without it the return type of the generator would be unknown to mypy